test/tests/functional/pbs_maintenance_reservations.py

```diff
@@ -244,7 +244,7 @@ def test_maintenance_delete(self):
         now = int(time.time())
 
         self.server.manager(MGR_CMD_SET, SERVER,
-                            {'managers': '%s@*' % TEST_USER})
+                            {'managers': (INCR, '%s@*' % TEST_USER)})
```

Comment:
This is fine, but why don't we just submit the reservation as MGR_USER if we want to submit it as a manager.  That way we don't have to add and delete TEST_USER from managers.

Reply:
Well, there was a bug in PBS wherein a user when acting as manager is allowed to submit a maintenance reservation but when the same user is taken off the manager's list, it was still able to delete the submitted reservation.
As you said, it can be done with MGR_USER too, in that case the test will be assuming that the user is already part of managers, it will then remove MGR_USER off the list of managers and then add it back again. I guess, Vaclav wanted the test to be more explicit and self contained.